repositories/order.go

```diff
@@ -3,8 +3,11 @@ package repositories
 import "Dp218Go/models"
 
 type OrderRepo interface {
-	CreateOrder(user models.User, scooter models.ScooterDTO) (models.Order, error)
-	SetOrderStart(order *models.Order, status models.ScooterStatusInRent) error
-	SetOrderEnd(order *models.Order, status models.ScooterStatusInRent) error
-	UpdateOrder(order *models.Order) error
+	CreateOrder(user models.User, scooterID, startID, endID int, distance float64) (models.Order, error)
+	GetAllOrders() (*models.OrderList, error)
+	//SetOrderStart(order *models.Order, status models.ScooterStatusInRent) error
```

Comment:
GetOrderByID() should also be very useful. In fact it's the only func, that I need for my purposes))